doc/develop/west/zephyr-cmds.rst

```diff
@@ -174,3 +174,25 @@ As does deleting them::
 Additionally the tool allows you to specify the modules you want to list,
 fetch or clean blobs for by typing the module names as a command-line
 parameter.
+
+.. _west-twister:
+
+Twister wrapper: ``west twister``
```

Comment:
Since people can override this, can we just call it `west test`. This way if someone wants to replace the test runner they can override the west command and call into their own. We can just document that by default the test runner is twister.

Reply:
> Since people can override this, can we just call it west test.

Wait, what? No, this is always going to use twister; this is not overridable

Reply:
ACK